src/inline-assembly.md

```diff
@@ -576,7 +585,10 @@ r[asm.options.checks.pure]
 - It is a compile-time error to specify `pure` on an asm block with no outputs or only discarded outputs (`_`).
 
 r[asm.options.checks.noreturn]
-- It is a compile-time error to specify `noreturn` on an asm block with outputs.
+- It is a compile-time error to specify `noreturn` on an asm block with outputs and without labels.
```

Comment:
```suggestion
- It is a compile-time error to specify `noreturn` on an asm block with outputs.
```

If it's a compile-time error to have a `label` block on an asm block with outputs (as below), then whether it's a compile-time error to specify `noreturn` on an asm block with outputs is invariant to whether there are labels, no?

Reply:
This change is to make `r[asm.options.checks.label-with-outputs]` be the only thing that blocks the `asm-goto-with-outputs` feature.

Reply:
Make sense, OK.